packages/zitadel-client/package.json

```diff
@@ -9,46 +9,40 @@
   "exports": {
     ".": {
       "types": "./dist/index.d.ts",
```

Comment:
I guess "types" can be removed on projects with esm only right?

Reply:
Honestly, I forgot if we can or not. I am unsure